app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/fragments/DialogPopupFragment.kt

```diff
@@ -1,27 +1,22 @@
-package edu.uc.hickmadc.customnotifications.ui.main
+package edu.uc.hickmadc.customnotifications.ui.main.fragments
 
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
 import androidx.fragment.app.DialogFragment
-import androidx.fragment.app.FragmentContainer
 import edu.uc.hickmadc.customnotifications.R
-import java.io.FileDescriptor
-import java.io.PrintWriter
-import java.util.zip.Inflater
 
 
-
-class dialogfragment: DialogFragment() {
+class DialogPopupFragment: DialogFragment() {
```

Comment:
Suggestion 2: I renamed dialogfragment to DialogPopupFragment to prevent future confusion. "dialogfragment" is a class made in this project, while "DialogFragment" is a class that's built-in to android studio. Additionally, classes are usually supposed to start with a capital letter in Kotlin coding conventions.
Sources: https://kotlinlang.org/docs/coding-conventions.html#naming-rules and https://developer.android.com/reference/androidx/fragment/app/DialogFragment

Reply:
Very good.